.github/workflows/ci-merge-coverage.yaml

```
@@ -12,10 +12,14 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 60
     steps:
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```

Let's use latest checkout version